stdlib/Unicode/src/Unicode.jl

```diff
@@ -5,10 +5,14 @@ module Unicode
 export graphemes
 
 """
+    Unicode.normalize(s::AbstractString; kwargs...)
```

Comment:
```suggestion
    Unicode.normalize(s::AbstractString; keywords...)
```

The text later says this.

Also, should perhaps be listed second (since it's the less common form)?

Reply:
That's the one called if you provide only one argument, that's why I put it first. But I don't have a strong opinion.